services/QuillLMS/spec/controllers/accessibility_settings_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AccessibilitySettingsController, type: :controller do
+  let(:user) { create(:user) }
+
+  before { allow(controller).to receive(:current_user) { user } }
+
+  describe 'GET #show' do
+    context 'when the user has no AccessibilitySetting yet' do
+      it 'returns null snackbar_duration' do
+        get :show, format: :json
+        expect(response).to have_http_status(:ok)
+        body = JSON.parse(response.body)
```

Comment:
This line comes up often enough that I think you would benefit from defining this in a `let` at the top of the file right under `let(:user)`.  `let(:body) { JSON.parse(response.body) }` will not actually call the parse block until the first reference in an `it` block, so this should work seemlessly and you won't need to define `body` in every `it` block.